requirements.txt

```diff
@@ -1,6 +1,9 @@
 # Requirements which are not versioned.
 defusedxml
 distro
+## flake8 6 removed --diff option which breaks flake precommit hook.
+## https://github.com/pycqa/flake8/issues/1389 https://github.com/PyCQA/flake8/pull/1720
+flake8<6.0.0
```

Comment:
@ashleypittman Should this one actually be in `cq/requirements.txt`?

Reply:
Looks like `cq` is for pylint only? I don't have strong option on this and open to either way.

Reply:
Separately I think we should use black formatter instead going forward, as flake8 has removed `--diff`.

Reply:
CQ is for any code quality tools and I thought these were already added there?

Reply:
I see. This one has been merged. Will move flake8 and yamlint to cq with my next PR.

Reply:
I was out last week so not able to check but I really thought these packages were already in that file, I'm not sure why they weren't.

I'm not familiar with the black formatter, it could be something to look at if you think it will help and has the right options.